server/session/player.go

```diff
@@ -93,19 +93,38 @@ const (
 )
 
 const (
-	containerArmour         = 6
-	containerChest          = 7
-	containerBeacon         = 8
-	containerFullInventory  = 12
-	containerCraftingGrid   = 13
-	containerHotbar         = 27
-	containerInventory      = 28
-	containerOffHand        = 33
-	containerBarrel         = 57
-	containerCursor         = 58
-	containerCreativeOutput = 59
+	containerArmour            = 6
+	containerChest             = 7
+	containerBeacon            = 8
+	containerFullInventory     = 12
+	containerCraftingGrid      = 13
+	containerFurnaceFuel       = 23
+	containerFurnaceResult     = 25
+	containerFurnaceInput      = 24
+	containerHotbar            = 27
+	containerInventory         = 28
+	containerOffHand           = 33
+	containerBlastFurnaceInput = 44
+	containerSmokerInput       = 45
+	containerBarrel            = 57
+	containerCursor            = 58
+	containerCreativeOutput    = 59
 )
 
+// smelter is an interface representing a block used to smelt items.
+type smelter interface {
+	// Durations returns the remaining, maximum, and cook durations of the smelter.
+	Durations() (time.Duration, time.Duration, time.Duration)
+	// UpdateDurations updates the remaining, maximum, and cook durations of the smelter.
+	UpdateDurations(remaining, max, cook time.Duration)
+	// Experience returns the collected experience of the smelter.
+	Experience() int
+	// SetExperience sets the collected experience of the smelter to the given value.
+	SetExperience(xp int)
+	// ResetExperience resets the collected experience of the smelter, and returns the amount of experience that was reset.
+	ResetExperience() int
```

Comment:
Remove methods from here that are not used. There's no point having methods in an interface if they're not used.

Reply:
Removed all unused methods.